starknet_py/tests/e2e/contract_interaction/interaction_test.py

```diff
@@ -54,40 +53,36 @@ async def test_throws_on_both_max_fee_and_auto_estimate(map_contract):
     value = 3
 
     invocation = map_contract.functions["put"].prepare(key, value)
-    with pytest.raises(ValueError) as exinfo:
+    with pytest.raises(
+        ValueError,
+        match="Max_fee and auto_estimate are exclusive and cannot be provided at the same time.",
```

Comment:
I think we wanted to start exceptions with "Arguments x and y...", right?

Reply:
Agreed. I can change it, please be aware that this is a another breaking change in Contract.

Reply:
I'll change it in new Account. Errors in Contract are already being addressed in #616 so it doesn't make sense to duplicate this work.